cli/types.go

```diff
@@ -0,0 +1,68 @@
+package cli
+
+import (
+	"github.com/flashbots/mev-boost/server"
+	"strings"
+	"net/url"
+	"errors"
+)
+
+var errDuplicateEntry = errors.New("duplicate entry")
+
+type relayList []server.RelayEntry
+
+func (r *relayList) String() string {
+	return strings.Join(server.RelayEntriesToStrings(*r), ",")
+}
+
+func (r *relayList) Contains(relay server.RelayEntry) bool {
+	for _, entry := range *r {
+		if relay.String() == entry.String() {
+			return true
+		}
+	}
+	return false
+}
+
+func (r *relayList) Set(value string) error {
+	relay, err := server.NewRelayEntry(value)
+	if err != nil {
+		return err
+	}
+	if r.Contains(relay) {
+		return errDuplicateEntry
```

Comment:
how about making this function idempotent, by not returning an error on duplicates?